### ISSUE TITLE
How to include module, function and line numbers.

### ISSUE THREAD

**Issue by james-ankura** [ConnorRigby/logger_backend_sqlite#1]
The fields are empty. I tried using metadata, but still empty.

**Comment by ConnorRigby** [comment]
All of that data should be supplied by Logger calls. Can you post your logger call?

**Comment by james-ankura** [comment]
Logger.debug "this is my message"

I have a simple logger dumper module that shows
```
{:debug, 
"INCOMING \"onblur\" on \"never_die\" to NeverDie.Channel\n  Transport:  Phoenix.Transports.WebSocket\n  Parameters: %{}", 
{{2019, 4, 4}, {14, 54, 27, 690}}, 
[
  pid: #PID<0.1645.0>, 
  line: 125, function: "channel_log/3", module: Phoenix.Logger,
  file: "lib/phoenix/logger.ex", application: :phoenix
]
}
```

But the sqlite3 table is missing: module, function, file, line, registered_name.

My config is 
```
config :logger, [
  utc_log: true,
  handle_otp_reports: true,
  handle_sasl_reports: true,
  backends: [LoggerBackendSqlite, {NotifierLogger, :notifier_logger}]
]

config :logger, :notifier_logger,
  level: :debug

config :logger, LoggerBackendSqlite,
  metadata: :all,
  database: 'debug_logs.sqlite3',
  max_logs: 9000 # defaults to 1000
```

**Comment by ConnorRigby** [comment]
Ah i wonder if the `Phoenix.Logger` does weird things that aren't quite the same as regular `Logger`. I can look into this. 

When you do a regular `Logger.debug("abc")` in your own code, does it work right?

**Comment by james-ankura** [comment]
Correct. You can see the output above. How do you get data on the table fields for module, function, etc?

**Comment by ConnorRigby** [comment]
Whoops i see i've made a copy paste typo. Fixing it right now!

**Comment by ConnorRigby** [comment]
Okay i've released 2.3.0 which should fix your issue. Thanks for the help debugging and point it out.
